client/src/menu/game/gameScreenContent/ChatMenu.tsx

```diff
@@ -76,6 +81,10 @@ function ChatMessageSection(): ReactElement {
             }
         }
 
+        if (GAME_MANAGER.state.stateType === "game") {
+            setMessages(GAME_MANAGER.state.chatMessages)
+        }
+
```

Comment:
What is this? Does it fix #469?

Reply:
This resolves a different issue that we never made an issue for

When the chat menu is first loaded it doesn't always show the messages. Now it does

Reply:
Can you check if it solves #469 tho? It looks like it should